internal/packages/archetype/_static/package-manifest.yml.tmpl

```diff
@@ -72,6 +72,22 @@ policy_templates:
           - localhost
 {{ end }}
 {{ end -}}
+{{ if .Manifest.Elasticsearch }}
+elasticsearch:
+{{ if .Manifest.Elasticsearch.SourceMode }}
+  source_mode: {{ .Manifest.Elasticsearch.SourceMode }}
+{{- end}}
+{{ if .Manifest.Elasticsearch.IndexMode }}
+  index_mode: {{ .Manifest.Elasticsearch.IndexMode }}
+{{- end }}
```

Comment:
Currently, there are no questions for these values when creating the package. So, right now they would not be written here in this manifest.

Should they be added also as part of the questions?

Reply:
Not sure, I could add them in a follow up if necessary. Or would you prefer the questions to be added here?

Reply:
Sure, they can be added in a follow up, it's not needed to do it now.